src/Features/Core/SolutionCrawler/WorkCoordinator.SemanticChangeProcessor.cs

```diff
@@ -382,14 +382,14 @@ protected override Task ExecuteAsync()
                         if (project == null)
                         {
                             data.AsyncToken.Dispose();
-                            return Task.FromResult(true);
+                            return SpecializedTasks.EmptyTask;
```

Comment:
It looks like a non-equivalent replacement. Are we sure the return value of type `Task` is never downcast to `Task<bool>` by a caller?